Day and Time/index.js

```diff
@@ -1,4 +1,25 @@
-function currentpage();
-{
-    window.print();
-}
\ No newline at end of file
+let date = new Date();
+console.log(date);
+let yr = date.getFullYear();
```

Comment:
Yr doesn't represent what it actually means. So lets write our variables clearly, in this case `year`